power_measurement.adoc

```diff
@@ -111,93 +107,110 @@ If you are measuring the performance of an experimental framework or system, you
 
-There are multiple phases to a benchmark as listed in the MLPerf
-Inference Rules document. Power measured is evaluated only on the
-performance phase of the benchmark and not in any other phases. To
-determine this exact section, Loadgen has been instrumented to indicate
-the start and stop of the performance phase of the benchmark and all
-power measurements are evaluated within this phase from the power
-logging done as part of the benchmark.
+https://github.com/mlcommons/inference_policies/blob/master/inference_rules.adoc[The MLPerf Inference rules]
+specify several phases of a benchmark: accuracy, performance, compliance.
+Power is evaluated only in the performance phase, and not in any other phases.
 
-The submission process has to use the software flow and scripts
-developed as part of the MLPerf benchmark Power measurement. The
-infrastructure has been developed by the MLPerf Power working group.
+=== LoadGen and Start/End TimeStamps
 
-=== Audit process and Benchmark Logging
+The MLPerf Power workflow uses exactly the same LoadGen as used for performance runs.
+LoadGen logs the system timestamp at the start and at the end of a performance run.
+The workflow then uses these timestamps to evaluate the power consumption of the run.
 
-As part of the submissions and logging, all the logs generated by the
-MLPerf Power SW infrastructure need to be submitted. These include the
-power meter ranging logs and the power measurement logs that are
-generated during the performance runs.
+=== Power and Performance Measurement and Reporting
 
-=== Replicability is mandatory
+Power and performance measurements should be from the same run for a
+given benchmark and scenario. The MLPerf Power workflow takes care of this by
+default. This must not be changed. Example: It is not permitted to run
+the same benchmark and scenario 3 times and report the highest
+performance and the lowest power consumption among the 3 runs.
 
-Results that cannot be replicated are not valid results.
+=== Realistic Power Management
 
-=== Power and Performance Reporting and Measurement
+The goal of the testing is to mimic real-world usage scenarios as much
+as possible and enable showing the benefits of realistic power
+management. Therefore, we require that:
 
-Power and performance measurements should be from the same run for a
-given benchmark and scenario. The current script takes care of this by
-default and it cannot and should not be changed. Example: We cannot run
-the same benchmark and scenario 3 times and report the highest
-performance and lowest power among the 3 runs.
+* Any power management system be qualified for use appropriate for the submission type (e.g., a generally available system must use software/firmware qualified for general availability and shipping with the platform).
+* No benchmark- or benchmarking-specific hacks.
+* Any changes in power management behavior must not have manual intervention or have awareness of the benchmark.
```

Comment:
@psyhtest NV-based platforms change power limits and QCOM-based platforms change frequency limits based on the benchmark. Assuming this will still be an acceptable practice going forward, could this proposed statement be viewed as being in contradiction?

Reply:
Agree.

Reply:
Removed "or have awareness of the benchmark" after consulting with the Power WG.